index.js

```diff
@@ -940,6 +945,8 @@ HTTP_LIGHTBULB.prototype = {
 
                 if (this.colorTemperature.unit === TemperatureUnit.KELVIN)
                     colorTemperature = Math.round(1000000 / colorTemperature); // converting Kelvin to mired
+                if (this.colorTemperature.unit === TemperatureUnit.PERCENT)
+                    colorTemperature = Math.round(400 - (3.5 * colorTemperature)); // converting percent to mired
```

Comment:
You are assuming a maxValue of 400 and a maxValue of 50 here. Would make more sense to make it dynamically adjust to the defined min/max values defined in the hap characteristic properties. 🤔 
Also is there are particular reasons for the mapping 0% -> 400 (aka warm white) and 100% maps to cold white? Seems to be pretty arbitrary and dependent on the scenario, why would somebody not endorse are mapping in the other way around 🤔

Reply:
The lightbulb I'm testing with seems to respond to this mapping (0% to maxValue and 100% to minValue) and plays nicely with the Home app's color wheel. However I do realize that some bulbs may have the opposite behavior, in which case switching min and max would work. Currently, minValue > maxValue is not allowed (which makes total sense). However, I could introduce another config option (possibly as another united, say "PERCENT_ALT" to map 0% to minValue and 100% to maxValue. This way you wouldn't have to break existing code. Let me know if that makes sense.